docs/3.0/upgrade.md

```diff
@@ -12,6 +12,22 @@ In order to update all Avo packages you should run the following command.
 bundle update avo avo_pro avo_advanced avo_dashboards avo_filters avo_menu avo_upgrade
 ```
 
+## Upgrade from 3.0.0.pre14 to 3.0.0.pre15
+:::option Avo::Current.current_user deprecation
+`Avo::Current.current_user` has been deprecated in favor of `Avo::Current.user`. Please ensure that you update this change in your project, as it will become obsolete in future versions.
+:::
+
+:::option Package dependencies reorganization
+
+We have made some enhancements to the way we manage package dependencies. Now, each tier should declare only one gem in the gemfile.
+
+For the community version, you only need to include `gem "avo"`.<br>
+For the pro tier, include only `gem "avo-pro"`<br>
+For the advanced tier, only the `gem "avo-advanced"` is required.
```

Comment:
Can we add the snippet we had on the initial ticket?
That was exactly how they would use it in the `Gemfile` with the source option too.